tutor/views.py

```diff
@@ -114,28 +103,24 @@ def grader(request):
             current_lesson_set, _, current_lesson, _, is_alternate = assignment.get_user_lesson(current_user.id)
             print("lessons in set:", current_lesson_set.lessons())
             print("my lesson:", current_lesson)
-            # Get submitted answer. No 'Confirm', no spaces, each ends w/ a semicolon
+            # Get submitted answer. No 'Confirm', no spaces, each ends with a semicolon
             submitted_answer = re.findall("Confirm [^;]*;|ensures [^;]*;", data['code'])
             submitted_answer = ''.join(submitted_answer)
 
-            # Send it off to the RESOLVE verifier (commented out because RESOLVE is down currently). You will need
-            # to import these 3 things: asyncio (not the django.asyncio), overall_status, and send_to_verifier
-            # response, vcs = asyncio.run(send_to_verifier(data['code']))
-            # if response is not None:
-            #     lines = overall_status(response, vcs)
-            #     status = response['status']
-            # else:
-            #     status = 'failure'
-            #     lines = []
-
-            # Placeholder for verifier response
-            status = 'success'
-            # issue: Eventually uncomment line below when lines are implemented in
-            # lines = [3, 5]
+            response, vcs = asyncio.run(send_to_verifier(data['code']))
+            if response is not None:
+                lines = overall_status(response, vcs)
+                status = response['status']
+            else:
+                status = 'failure'
+                lines = []
 
             # Log data
             log_data(current_user, assignment, current_lesson_set, current_lesson, is_alternate, data, status)
 
+            # issue: Eventually send these line nums to the browser to highlight the lines that passed/failed
+            print("Line nums that will eventually be sent to browser:", lines)
```

Comment:
You may need to comment this out for now.

Reply:
Done!